server/graphql/v2/input/MemberInvitationReferenceInput.ts

```diff
@@ -0,0 +1,41 @@
+import { GraphQLInputObjectType, GraphQLInt, GraphQLString } from 'graphql';
+
+import models from '../../../models';
+import { NotFound } from '../../errors';
+import { idDecode, IDENTIFIER_TYPES } from '../identifiers';
+
+/**
+ * An input for referencing MemberInvitations.
+ */
+export const MemberInvitationReferenceInput = new GraphQLInputObjectType({
+  name: 'MemberInvitationReferenceInput',
+  fields: () => ({
+    id: {
+      type: GraphQLString,
+      description: 'The public id identifying the member invitation (ie: dgm9bnk8-0437xqry-ejpvzeol-jdayw5re)',
+    },
+    legacyId: {
+      type: GraphQLInt,
+      description: 'The internal id of the invitation (ie: 580)',
+    },
+  }),
+});
+
+export const fetchMemberInvitationWithReference = async (
+  input,
+  { throwIfMissing } = { throwIfMissing: false },
+): Promise<any> => {
+  let MemberInvitation;
```

Comment:
Variable names default to camel case, I'm surprised that ESLint didn't pick this

```suggestion
  let memberInvitation;
```

Reply:
Hmm, it looks like ESLint doesn't check the first character - [Online Demo](https://eslint.org/demo#eyJ0ZXh0IjoiLyplc2xpbnQgY2FtZWxjYXNlOiBcImVycm9yXCIqL1xuXG5sZXQgQ2FtZWxDYXNlID0gMTA7XG5sZXQgQ2FtZWxfQ2FzZSA9IDEwO1xuIiwib3B0aW9ucyI6eyJwYXJzZXJPcHRpb25zIjp7ImVjbWFWZXJzaW9uIjoxMiwic291cmNlVHlwZSI6InNjcmlwdCIsImVjbWFGZWF0dXJlcyI6e319LCJydWxlcyI6e30sImVudiI6e319fQ==),  might be a bug I will report.